FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.SchemaManagement/Parquet/FhirParquetSchemaManager.cs

```diff
@@ -83,9 +90,74 @@ public FhirParquetSchemaNode GetSchema(string schemaType)
             return _resourceSchemaNodesMap[schemaType];
         }
 
+        public Dictionary<string, string> GetAllSchemaContent()
+        {
+            return _resourceSchemaNodesMap.ToDictionary(pair => pair.Key, pair => Newtonsoft.Json.JsonConvert.SerializeObject(pair.Value));
+        }
+
         public Dictionary<string, FhirParquetSchemaNode> GetAllSchemas()
         {
             return new Dictionary<string, FhirParquetSchemaNode>(_resourceSchemaNodesMap);
         }
+
+        private Dictionary<string, FhirParquetSchemaNode> LoadSchemas(string schemaDirectoryPath)
```

Comment:
Currently the local default schemas have become embedded resource?

Reply:
Thanks. Removed